tests/test_repository.py

```diff
@@ -16,7 +16,13 @@
 from twine import repository
 from twine.utils import DEFAULT_REPOSITORY
 
+try:
+    from unittest import mock
```

Comment:
This project doesn't use mock. Instead we use pretend which is already imported below and used throughout the test file. Please update your test to use that instead.

Reply:
Turns out I didn't need mock after all. Just replaced ``ProgressBar`` with a context manager and it did the trick. Thanks!